projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -3008,7 +3008,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
     /**
      * @hidden @internal
      */
-    private get hasNoData(): boolean {
+    public get hasNoData(): boolean {
```

Comment:
Since `hasNoData` will not be used in the template, keep its accessor to be private.